website/src/content/docs/internals/changelog.md

```diff
@@ -21,10 +21,18 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
 ### CLI
 
+#### Bug fixes
+
 - Fix configuration resolution. Biome is now able to correctly find the `biome.jsonc` configuration file when `--config-path` is explicitly set. Contributed by @Sec-ant
 
+- JavaScript/TypeScript files of different variants (`.ts`, `.js`, `.tsx`, `.jsx`) in a single workspace now have stable formatting behaviors when running the CLI command in paths of different nested levels or in different operating systems ([#2080](https://github.com/biomejs/biome/issues/2080), [#2109](https://github.com/biomejs/biome/issues/2109)). Contributed by @Sec-ant
+
 ### Configuration
 
+#### Bug fixes
+
+- Complete the documentation and overrides support for options `lineEnd` and `attributePosition`. Contributed by @Sec-ant
```

Comment:
It's `lineEnding`. Also, it's usually more helpful to show all the keys, e.g. `formatter.lineEnding` and `formatter.attributePosition`. Same as our documentation

Reply:
Fixed in 6d3fc21.